### PR TITLE
tech: fix some STAT vulnerabilities related to unreleased resources

### PR DESCRIPTION
**Pull request by pikachugb**
- resolved some unreleased streams by closing them using commons-io closeQuietly
- resolved path manipulation from ImageService by normalizing the path (https://matam-kirankumar.medium.com/fortify-issues-path-manipulation-3ac42c1d1e9a)

### CODE REVIEW

File: src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java [hunk 1]
```diff
@@ -58,7 +59,9 @@ public class ImageService {
     private File saveImageToTempFile(String pictureLink) throws Exception {
 
         String tmpPath = System.getProperty("java.io.tmpdir");
-        File tmpDir = new File(tmpPath);
+        String normalizedPath = FilenameUtils.normalize(tmpPath);
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java:62]: what's this?

File: src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/util/PropertyUtil.java [hunk 2]
```diff
@@ -101,11 +103,17 @@ private static Long getLong(PropertyKeys property) {
...
             try {
                 fileProps = new Properties();
-                fileProps.load(PropertyUtil.class.getClassLoader().getResourceAsStream(MAIN_CONFIG_FILE_NAME));
+                resourceStream = PropertyUtil.class.getClassLoader().getResourceAsStream(MAIN_CONFIG_FILE_NAME);
+
+                fileProps.load(resourceStream);
             } catch (IOException ex) {
                 throw new RuntimeException("Error occured while loading config file: " + MAIN_CONFIG_FILE_NAME + ", " + ex);
+            } finally {
+                IOUtils.closeQuietly(resourceStream);
```
Comment (vladbenchea) [inline src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/util/PropertyUtil.java:116]: if input is null here, doesn't it throw an error?

Comment (pikachugb) [reply src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/util/PropertyUtil.java:116]: it has a null check inside the method

**Review by vladbenchea — APPROVED** [review]

File: src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java (hunk 1 above)
Comment (pikachugb) [reply src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/ImageService.java:62]: it's written in the description

**Pull request merged**